modules/oe_webtools_etrans/src/Plugin/Block/UnifiedEtransBlock.php

```diff
@@ -306,4 +311,27 @@ protected function preparesWtEtransJson(string $translation_from, string $placeh
     return $json;
   }
 
+  /**
+   * Attempt to map langcode to ISO 639-2.
```

Comment:
```suggestion
   * Attempt to map langcode to Drupal language codes.
```

Reply:
No, it does map to ISO-639-1 (you're correct below. alpha-2 is 3 letters, alpha-1 is 2).